src/ui/public/vislib/visualizations/point_series/_point_seri.js

```diff
@@ -0,0 +1,87 @@
+import _ from 'lodash';
+import errors from 'ui/errors';
+
+export default function PointSeriProvider(Private) {
+
+  class PointSeri {
```

Comment:
Yeah, `series` is a troublesome word. What's the plural of `series`? `serieses`? it's a bit of a shock at first to me as well, but I think we might be better off just moving away from `series` entirely.